### ISSUE TITLE
Add SimpleSwitcher to Chocolatey Community Repository

### ISSUE THREAD

**Issue by AlexRadch** [Aegel5/SimpleSwitcher#31]
Add SimpleSwitcher to Chocolatey Community Repository, please. https://community.chocolatey.org/

**Comment by Aegel5** [comment]
Так как планируется добавить в саму программу возможность обновиться, то дублировать и поддерживать обновление через chocolatey смотрится слишком затратным.

**Comment by Aegel5** [comment]
Да, уже предлагали добавить в scoop.sh, и, в общем то, это хороший вариант, так как обновление вручную сделать сложно, но есть проблемы:
1) SS может работать от администратора и тогда scoop тоже нужно запускать от администратора, чтобы обновиться (не большая проблема, но все не очень удобно, можно забыть, например, про это).
2) scoop не умеет устанавливать в пользовательскую папку ( у них есть https://github.com/ScoopInstaller/Scoop/issues/3559 на эту тему, хотелось бы чтобы они добавили такую возможность).

**Comment by pongo** [comment]
У scoop в этом плане свой подход: все программы ставятся в папку scoop. И я бы не ожидал в обозримом будущем, что они добавят возможность выбора папки для установки. Для конфигов у них есть опция persist: конфиги находятся в отдельной папке, а scoop создает символическую ссылку на конфиг в папке с программой.

Как я понимаю, в SS даже менять ничего не придется — всё будет работать.

1. т.к. всё ставится в папку scoop, то для установки не нужно запускать scoop под админом. SS, после запуска, может увидеть в конфиге, что нужно запускаться под админом и вызвать UAC или перезапустить себя с UAC.
2. 
3. вызвать scoop install. конфиги остаются в папке scoop/persist

То есть если Chocolatey позволяет устанавливать программы куда угодно и как угодно, то scoop всё ставит у себя.
